authors/settings.py

```diff
@@ -136,7 +136,7 @@
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```

Comment:
Please Team,
Note that I commented out this part of the code because it was preventing tests from running 
Whoever wrote it should look into it

Reply:
We should discuss it as a team actually

Reply:
Have you figured why this was preventing tests from passing? @kafuuma